### PR TITLE
Add function to replace actor in ticket through the action menu

### PR DESCRIPTION
**Pull request by Drimmwald**
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

Hello!
I made some changes. Maybe it will be interesting for you.
"Add function to replace actor in ticket through the action menu"
I think it comfortably for users, faster change actor.

replace

### CODE REVIEW

**Comment by AnaelMobilia** [comment]
@Drimmwald Please don't include locales/* files : translation are made on transifex only (and files imported :-))

**Comment by orthagh** [comment]
@Drimmwald, It's a good feature !

Could you add changes required by @AnaelMobilia ?

Additionnaly:
- Add GPL headers when adding new files (take example from other php files of the project)
- Fix coding standard issues reported by travis: https://travis-ci.org/glpi-project/glpi/jobs/309362964#L1225

File: ajax/dropdownTicketCurrentActors.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                'rand'        => $rand,
+                'width'       => '150',
+                'ldap_import' => true);
+            
+            $actors = [];
+            foreach($_POST['tickets'] as $ticket_id) {
+               $ticket = new Ticket();
+               $ticket->getFromDB($ticket_id);
+
+               foreach($ticket->getTicketActorsByType('user') as $actor_id => $actor)
```
Comment (trasher) [inline ajax/dropdownTicketCurrentActors.php]: Avoid clling methods in a loop:
```php
$ticket_actors = $ticket->getTicketActorsByType('user');
foreach ($ticket_actors as $actor_id => $actor){
   // [...]
}
```

File: ajax/dropdownTicketCurrentActors.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Session::checkCentralAccess();
+global $CFG_GLPI;
+// Make a select box
+if (isset($_POST["type"])
+    && isset($_POST["actortype"])
+    && isset($_POST["itemtype"])) {
+   $rand = mt_rand();
+   if ($item = getItemForItemtype($_POST["itemtype"])) {
+      switch ($_POST["type"]) {
```
Comment (trasher) [inline ajax/dropdownTicketCurrentActors.php]: Some part of code seems very similar; this should be factorized.

File: inc/ticket.class.php [hunk 3]
```diff
@@ -7284,6 +7428,31 @@ function getTicketActors() {
       return $ticket_users_keys;
    }
 
+   function getTicketActorsByType($objtype = 'user') {
```
Comment (trasher) [inline inc/ticket.class.php]: Add phpdoc for methods that does not exists in parent class

**Comment by trasher** [comment]
ping?

**Comment by trasher** [comment]
See https://github.com/glpi-project/glpi/issues/1262

**Comment by trasher** [comment]
This PR is not OK, and author did not give feedback. Closing.

**Pull request closed without merging**